extensions-core/s3-extensions/src/main/java/org/apache/druid/storage/s3/output/S3OutputConfig.java

```diff
@@ -48,10 +48,7 @@
 
   @Nullable
   @JsonProperty
-  private HumanReadableBytes chunkSize;
```

Comment:
Please update the documentation also.
The release notes should mention that the defaults have changed and the value is removed.